stories/Accordion/AccordionGroupsExample.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { useState } from 'react';
```

Comment:
Shouldn't this be renamed to `CollapseGroupsExample` in the `Collapse` folder?

Reply:
I deleted the file it is not used